tempodb/backend/s3/config.go

```diff
@@ -25,12 +25,13 @@ type Config struct {
 	HedgeRequestsAt   time.Duration  `yaml:"hedge_requests_at"`
 	HedgeRequestsUpTo int            `yaml:"hedge_requests_up_to"`
 	// SignatureV2 configures the object storage to use V2 signing instead of V4
-	SignatureV2      bool              `yaml:"signature_v2"`
-	ForcePathStyle   bool              `yaml:"forcepathstyle"`
-	BucketLookupType int               `yaml:"bucket_lookup_type"`
-	Tags             map[string]string `yaml:"tags"`
-	StorageClass     string            `yaml:"storage_class"`
-	Metadata         map[string]string `yaml:"metadata"`
+	SignatureV2          bool              `yaml:"signature_v2"`
+	ForcePathStyle       bool              `yaml:"forcepathstyle"`
+	BucketLookupType     int               `yaml:"bucket_lookup_type"`
+	Tags                 map[string]string `yaml:"tags"`
+	StorageClass         string            `yaml:"storage_class"`
+	Metadata             map[string]string `yaml:"metadata"`
+	NativeAWSAuthEnabled bool              `yaml:"native_aws_auth_enabled"`
```

Comment:
should we mimic the thanos config name? since that's probably what it's called in mimi/loki?

Reply:
I copied the name from [`mimir`](https://github.com/grafana/mimir/blob/main/pkg/storage/bucket/s3/config.go#L87). In `thanos-io/objstore` is called [`aws_sdk_auth`](https://github.com/thanos-io/objstore/blob/main/providers/s3/s3.go#L128). I don't have a preference.

Reply:
Ah, didn't realize that this name came from Mimir. Good call :+1: